packages/lib/src/alert/Alert.tsx

```diff
@@ -157,91 +156,94 @@ export default function DxcAlert({
     if (currentIndex === messages.length) handlePrevOnClick();
   }, [currentIndex, messages, handlePrevOnClick]);
 
+  useEffect(() => {
```

Comment:
To ease the component load a bit, does it make sense to move this `useEffect` to the `ModalAlertWrapper` component?

Reply:
In any case, to avoid unnecessary re-renders, we need to wrap `handleOnClose` in a `useCallback`.